### PR TITLE
move result number to All search result pill

### PR DESCRIPTION
**Pull request by w8896699**
## Summary
delete the pill shows all result under search bar, add total search number in All search result pill

#903

### CODE REVIEW

File: app-web/src/utils/helpers.js [hunk 1]
```diff
@@ -61,6 +61,20 @@ export const getTextAndLink = (resourceType, resourcesByType) => {
...
+/** return a combine text result or results depens on how much search results we have
+ * @param {num} numOfResults
+ * @returns {string}
+ */
+export const getSearchResultLable = numOfResults => {
+  let resultLabel;
+  if (numOfResults) {
+    resultLabel = `${numOfResults} Result${numOfResults > 1 ? 's' : ''}`;
+  } else {
+    resultLabel = `No Search Result`;
```
Comment (w8896699) [inline app-web/src/utils/helpers.js]: This else loop relative to a search bug with github search result,  Patrick will fix it in his pr and I will remove this loop afterward.

File: app-web/src/components/Home/ResourcePreview.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ import { Container, LinkContainer } from './index';
 import Card from '../Cards/Card/Card';
 import Pill from '../UI/Pill';
 import { RESOURCE_TYPES } from '../../constants/ui';
+import { getSearchResultLable } from '../../utils/helpers';
```
Comment (sheaphillips) [inline app-web/src/components/Home/ResourcePreview.js]: mis-spelling here - should be "label" not "lable"

File: app-web/src/utils/helpers.js (hunk 1 above)
Comment (sheaphillips) [reply app-web/src/utils/helpers.js]: ok.

File: app-web/src/components/Home/ResourcePreview.js (hunk 2 above)
Comment (w8896699) [reply app-web/src/components/Home/ResourcePreview.js]: Thanks, I will fix this in my next pr.

File: app-web/src/utils/helpers.js (hunk 1 above)
Comment (patricksimonian) [reply app-web/src/utils/helpers.js]: 'No Search Results'

**Pull request merged**